javav2/example_code/sns/Readme.md

```diff
@@ -1,6 +1,52 @@
+- [Sending an Amazon SNS text message.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/PublishTextSMS.java) (Publish command)
+- [Publishing an Amazon SNS topic.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/PublishTopic.java) (Publish command)
+- [Setting attributes for Amazon SNS.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/SetSMSAttributes.java) (SetSMSAttributes command)
+- [Setting attributes for Amazon SNS topic.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/SetTopicAttributes.java) (SetTopicAttributes command)
+- [Subscribing to an Amazon SNS email endpoint.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/SubscribeEmail.java) (Subscribe command) 
+- [Subscribing to an Amazon SNS HTTPs endpoint.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/SubscribeHTTPS.java) (Subscribe command) 
+- [Subscribing to an Amazon SNS lambda function.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/SubscribeLambda.java) (Subscribe command) 
+- [Subscribing to an Amazon SNS text endpoint.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/SubscribeTextSMS.java) (SubscribeTextSMS command)  
+- [Removing an Amazon SNS subscription.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/Unsubscribe.java) (Unsubscribe command)  
+- [Initializing and using the example SNSMessageFilterPolicy class.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/UseMessageFilterPolicy.java) (Apply command)  
+
+### Cross-service examples
+
+- [Build an application to submit data to a DynamoDB table](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/usecases/creating_first_project) (Multiple commands)
+- [Build a publish and subscription application that translates messages](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/usecases/creating_sns_sample_app) (Multiple commands)
+- [Build a publish and subscription application that translates messages using Java Async API](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/usecases/creating_sns_async) (Multiple commands)
+- [Use scheduled events to invoke a Lambda function](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/usecases/creating_scheduled_events) (Multiple commands)
```

Comment:
Most of the Single action link titles start with a gerund (verb ending in "ing"), but these don't.

Reply:
fixed